src/v2_5/user.rs

```diff
@@ -6,10 +6,22 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+use serde_utils;
+use super::data::Data;
+
 #[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
 pub struct User {
+    // A DSP's UID for this user (see user matching for format).
     #[serde(rename = "buyeruid", skip_serializing_if = "Option::is_none")]
     pub buyer_uid: Option<String>,
+    // This object used by publishers to pass additional attributes about the user or content.
+    #[serde(skip_serializing_if = "Vec::is_empty")]
+    pub data: Vec<Data>,
+    // The User Ext Object, which is used to indicate requests that contain certain user identifiers and are subject to GDPR regulations.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub ext: Option<serde_utils::Ext>,
```

Comment:
same here. not an issue but it`s prettier to add the `ext` field as  last

Reply:
as specs